cuenca/resources/bill_payments.py

```diff
@@ -0,0 +1,25 @@
+from dataclasses import dataclass
+from typing import ClassVar, Optional, cast
+
+from cuenca_validations.types import BillPaymentQuery
+
+from .base import Transaction
+from .resources import retrieve_uri
+from .service_providers import ServiceProvider
+
+
+@dataclass
+class BillPayment(Transaction):
+    _resource: ClassVar = 'bill_payment'
+    _query_params: ClassVar = BillPaymentQuery
+
+    account_number: str
+    provider_uri: Optional[str]
```

Comment:
how is it possible for this to be none?

Reply:
First idea was to deploy BillPayments first and then ServiceProviders, so the `Optional` was to have into account that a `ServiceProvider` uri might not be available.
Now we also are going to deploy them together so this might not be the right approach.